КР/PriorityQueue/PriorityQueue/PriorityQueue.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace PriorityQueue;
+
+public class PriorityQueue<T> where T : IPriority
+{
+        public LinkedList<T> Entries { get; } = new LinkedList<T>();
```

Comment:
```suggestion
        public LinkedList<T> Entries { get; } = new();
```
И кажется, что ему незачем быть public-свойством.